README.md

```diff
@@ -1,31 +1,34 @@
 # nRF Connect Power Profiler
 
-Power Profiler app for
-[nRF Connect](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher).
-
-## Introduction
-
-_nRF Connect Power Profiler_ is a tool to communicate with the
+nRF Connect Power Profiler is a tool to communicate with the
 [Power Profiler Kit II (PPK2)](https://www.nordicsemi.com/Software-and-tools/Development-Tools/Power-Profiler-Kit-2),
 an affordable and flexible tool to obtain real-time current measurements of your
-designs. It also supports the
-[older, first version of the Power Profiler Kit](https://www.nordicsemi.com/Software-and-tools/Development-Tools/Power-Profiler-Kit).
+designs.
+
 The PPK measures current consumption for a connected Nordic Development Kit or
 any external board. It gives a detailed picture of the current profile for the
 user application.
 
-All functionality is described in the
-[User Guide](https://infocenter.nordicsemi.com/topic/ug_ppk2/UG/ppk/PPK_user_guide_Intro.html)
-(If you use the first version of the hardware, there is a different
-[User Guide for the PPK 1](https://infocenter.nordicsemi.com/topic/ug_ppk/UG/ppk/PPK_user_guide_Intro.html)).
-
 ![screenshot](resources/screenshot.png)
 
+The application also supports the
+[older, first version of the Power Profiler Kit](https://www.nordicsemi.com/Software-and-tools/Development-Tools/Power-Profiler-Kit).
+
 ## Installation
 
-See the
-[InfoCenter](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_nrftools%2Fstruct%2Fnrftools_nrfconnect.html)
-pages for information on how to install the application.
+nRF Connect Power Profiler is installed from nRF Connect from Desktop. For detailed
+steps, see
+[Installing nRF Connect for Desktop apps](https://docs.nordicsemi.com/bundle/nrf-connect-desktop/page/installing_apps.html)
+in the nRF Connect from Desktop documentation.
+
+## Documentation
+
+Read the
+[nRF Connect Power Profiler](https://docs.nordicsemi.com/bundle/nrf-connect-ppk/page/index.html)
+official documentation for information about its UI and features.
+
+(The documentation is valid for PPK2. If you use the first version of the hardware, see the
+[online documentation for PPK1](https://docs.nordicsemi.com/bundle/ug_ppk/page/UG/ppk/PPK_user_guide_Intro.html)).
```

Comment:
To be removed PPK 1 deprecated

Reply:
I'll keep this line as it is informative about what the scope of the docs is.